### DISCUSSION TITLE
Can Strimzi Kafka and Consul be integrated via Consul agents?

### DISCUSSION THREAD

**Discussion by seemasanjaisinghani** [strimzi/strimzi-kafka-operator#6142] [Q&A]
Can we strimzi kafka and consul be integrated for service discovery via consul agents? In kubernetes?

**Comment by seemasanjaisinghani** [comment]
@scholzj Any information I can refer for this ?

**Reply by scholzj** [reply]
Strimzi certainly does not have any integration with Consul. And TBH, I never used it and not sure what exactly you are asking for - so I do not know if it is possible or not.
